### PR TITLE
README-dev: add TOC

### PR DESCRIPTION
**Pull request by dougsland**
Adding Table of Content to help navigate.

Previously: https://github.com/containers/hirte/pull/305 - now a clean PR.

### CODE REVIEW

**Comment by dougsland** [comment]
The failure not seems related to the error:
```
23:45:20                 out: >       assert bar.wait_for_unit_state_to_be(hirte_dep_service, "inactive")
23:45:20                 out: E       AssertionError: assert False
23:45:20                 out: E        +  where False = <bound method HirteContainer.wait_for_unit_state_to_be of <hirte_test.container.HirteNodeContainer object at 0x7f4f6b6b9f00>>('hirte-dep@simple.service', 'inactive')
23:45:20                 out: E        +    where <bound method HirteContainer.wait_for_unit_state_to_be of <hirte_test.container.HirteNodeContainer object at 0x7f4f6b6b9f00>> = <hirte_test.container.HirteNodeContainer object at 0x7f4f6b6b9f00>.wait_for_unit_state_to_be
23:45:20                 out: 
23:45:20                 out: test_proxy_service_fails_on_execstart.py:27: AssertionError
23:45:20                 out: =========================== short test summary info ============================
23:45:20                 out: FAILED test_proxy_service_fails_on_execstart.py::test_proxy_service_fails_on_execstart - AssertionError: assert False
23:45:20                 out:  +  where False = <bound method HirteContainer.wait_for_unit_state_to_be of <hirte_test.container.HirteNodeContainer object at 0x7f4f6b6b9f00>>('hirte-dep@simple.service', 'inactive')
23:45:20                 out:  +    where <bound method HirteContainer.wait_for_unit_state_to_be of <hirte_test.container.HirteNodeContainer object at 0x7f4f6b6b9f00>> = <hirte_test.container.HirteNodeContainer object at 0x7f4f6b6b9f00>.wait_for_unit_state_to_be
23:45:20                 out: ============================== 1 failed in 5.94s ===============================
23:45:20         Command event: 6.383 waiting for stream readers
```

/cc @engelmi

File: README.developer.md
Comment (engelmi) [inline README.developer.md]: I think we can omit the `Hirte` entry here.

**Review by engelmi — APPROVED** [review]
LGTM
I have just a minor comment, which would be great to be resolved before merging.

File: README.developer.md
Comment (mwperina) [reply README.developer.md]: +1

**Comment by dougsland** [comment]
made the changes but let's hold for merning https://github.com/containers/hirte/pull/313 first. It's more complicated to rebase than this trivial patch.

/hold

**Comment by pypingou** [comment]
Looks like the markdown linter has a few things to say :)

**Comment by dougsland** [comment]
> Looks like the markdown linter has a few things to say :)

yep :P Going to fix soon, sorry.

**Comment by dougsland** [comment]
looks like the failure is not related:

```
19:21:36                 out: test_proxy_service_fails_on_execstart.py:27: AssertionError
19:21:36                 out: =========================== short test summary info ============================
19:21:36                 out: FAILED test_proxy_service_fails_on_execstart.py::test_proxy_service_fails_on_execstart - AssertionError: assert False
19:21:36                 out:  +  where False = <bound method HirteContainer.wait_for_unit_state_to_be of <hirte_test.container.HirteNodeContainer object at 0x7f54e9bf1f00>>('hirte-dep@simple.service', 'inactive')
19:21:36                 out:  +    where <bound method HirteContainer.wait_for_unit_state_to_be of <hirte_test.container.HirteNodeContainer object at 0x7f54e9bf1f00>> = <hirte_test.container.HirteNodeContainer object at 0x7f54e9bf1f00>.wait_for_unit_state_to_be
```

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**
